runtime/rococo/src/migration/P9100.rs

```diff
@@ -31,7 +31,7 @@ where
 	<T as frame_system::Config>::Event: From<pallet_parachain_staking::Event<T>>,
 {
 	#[cfg(feature = "try-runtime")]
-	fn pre_upgrade() -> Result<(), &'static str> {
+	fn pre_upgrade() -> Result<Vec<u8>, &'static str> {
```

Comment:
So these files are not used and only keep for history backup purposes. If no CI error, maybe we should just leave it?

I am open to any solution. minor issue.